Updates to CUDAService from Salvatore

makortel opened the pull request:
This PR adds fixes and additional developments for CUDAService testing by @diguida.

Tested in 10_1_0.

makortel commented on HeterogeneousCore/CUDAUtilities/interface/getCudaDrvErrorString.h:
```diff
@@ -0,0 +1,17 @@
+
+inline const char *getCudaDrvErrorString(CUresult error_id) {
+  const char *message;
+  auto ret = cuGetErrorName(error_id, &message);
```
This function was originally added in a6bc36ae6585fa9a14a2ee40862b3050b282795d with a "copy-pasted" array of error codes and their string counterparts, and then moved around in fa62ef125f57d2f3596c65409037932546d82a40. But while preparing this PR I noticed this
https://github.com/cms-patatrack/cmssw/blob/c0cd5117e0e553a4416324a73f5ac05f0621e901/HeterogeneousCore/CUDAUtilities/interface/cudaCheck.h#L14-L17
and realized that very likely the "copy-pasted" array can be replaced with a call to `cuGetErrorName()`. Nevertheless I think the function has some added value on top of `cuGetErrorName()` by returning the C-string.

What do others think?

Also is the current level of cleanup (in commits 88404a1f1db2cc26d7178346740419aac57b587d and c28a0848dea98263185f1ce7123fd2fd22d638e2) good-enough or would people prefer squashing them around (so that the "copy-pasted" array or `BuildFile.xml` edits do not show up in history)?

fwyzard replied:
I don't mind having the extra function around - if we end up not using it, we can drop it later on.

As for the commits history: I've been squashing most of the PR because they tended to have a long history of not-very-meaningful commits; IMHO the commit history in the PR should reflect the changes we want to make, not how we got there.

For this PR, I would keep more than one commit, but I would
  - squash 08c0dbf and ff8a8bf into f12bdf4
  - squash c28a084, 88404a1 and fa62ef1 into a6bc36a

Something like
```
pick    f12bdf4 Add an unit test for CUDAService.
squash  08c0dbf Improved CUDAService unit test: if the runtime API is working, the service is guaranteed to be enabled, and it runs the checks on the getters.
squash  ff8a8bf Minor fixes for testCUDAService unit test.

pick    06ff473 Added new CUDAService unit test getting information from the driver API.

pick    a6bc36a Improve Error handling in CUDAService.
squash  fa62ef1 Factor out CUDA driver API error string utility code into a standalone header on a dedicated package, and include it in CUDAService.
squash  88404a1 Use cuGetErrorName() instead of duplicating the error names
squash  c28a084 Remove unnecessary additions to BuildFiles

reword  a62a0d6 Do not dlopen libcuda.so: this is installation and/or ld.conf dependent, so just rely on the -l cuda in the linker, as set up by cuda-stubs external.

reword  a6ab6d9 Bug fix: if the CUDAService is disabled by configuration, the ctor just returns, so it does not set any data member. This has been found in the unit test by explicitly forcing the configuration in the CUDAService PSet to be disabled: the last assert fails.
```

fwyzard replied:
@makortel how would you like me to proceed ?
  - clean up the history, push the changes to your branch, and merge it (PR will show up as "merged")
  - same but leave your two commits un-squashed
  - clean up the commits locally and merge it (PR will show up as "closed")
  - same but leave your two commits un-squashed
  - merge the PR, then clean up the history (PR should still show up as "merged")
  - same but leave your two commits un-squashed

makortel replied:
@fwyzard I'll update the branch later today (sorry for delay...).

fwyzard commented on HeterogeneousCore/CUDAUtilities/interface/getCudaDrvErrorString.h:
```diff
@@ -0,0 +1,17 @@
+#ifndef HeterogeneousCore_CUDAUtilities_getCudaDrvErrorString_h
+#define HeterogeneousCore_CUDAUtilities_getCudaDrvErrorString_h
+
+#include "cuda.h"
```
```
#include <cuda.h>
```

makortel commented:
Fixed the `#include` and squashed commits together (along but not exactly as @fwyzard's suggestion https://github.com/cms-patatrack/cmssw/pull/38#discussion_r179109379).

fwyzard commented:
### /RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValTTbar_13-CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/10824.5/index.html) for workflow 10824.5
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValTTbar_13-CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/10824.8/index.html) for workflow 10824.8
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValTTbar_13-CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/10824.7/index.html) for workflow 10824.7

### /RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValZMM_13-CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/10824.5/index.html) for workflow 10824.5
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValZMM_13-CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/10824.8/index.html) for workflow 10824.8
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValZMM_13-CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/10824.7/index.html) for workflow 10824.7

### /RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - reference [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-74203703f0f911fed5d2697a665c5cd8177d84f2/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.7
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.7
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5%3A;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference workflow 10824.5
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=testing-c0cd5117e0e553a4416324a73f5ac05f0621e901//RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8%3A;referenceobj2=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.8
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=testing-c0cd5117e0e553a4416324a73f5ac05f0621e901//RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7%3A;referenceobj2=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.7

### /RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - reference [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-74203703f0f911fed5d2697a665c5cd8177d84f2/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.7
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.7
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5%3A;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference workflow 10824.5
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=testing-c0cd5117e0e553a4416324a73f5ac05f0621e901//RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8%3A;referenceobj2=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.8
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=testing-c0cd5117e0e553a4416324a73f5ac05f0621e901//RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7%3A;referenceobj2=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.7

fwyzard commented:
This is the report I generate with my validation script... do you think the comparisons make sense ?

fwyzard commented:
Belay that... looks like some of the DQM links are not yet working correctly.

fwyzard commented:
### /RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValTTbar_13-CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/10824.5/index.html) for workflow 10824.5
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValTTbar_13-CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/10824.8/index.html) for workflow 10824.8
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValTTbar_13-CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/10824.7/index.html) for workflow 10824.7

### /RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValZMM_13-CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/10824.5/index.html) for workflow 10824.5
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValZMM_13-CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/10824.8/index.html) for workflow 10824.8
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValZMM_13-CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/10824.7/index.html) for workflow 10824.7

### /RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - reference [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-74203703f0f911fed5d2697a665c5cd8177d84f2/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.7
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.7
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5%3A;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference workflow 10824.5
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8%3A;referenceobj2=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.8
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7%3A;referenceobj2=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.7

### /RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - reference [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-74203703f0f911fed5d2697a665c5cd8177d84f2/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.7
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.7
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5%3A;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference workflow 10824.5
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8%3A;referenceobj2=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.8
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7%3A;referenceobj2=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.7

fwyzard commented:
### /RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValTTbar_13-CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/10824.5/index.html) for workflow 10824.5
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValTTbar_13-CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/10824.8/index.html) for workflow 10824.8
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValTTbar_13-CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/10824.7/index.html) for workflow 10824.7

### /RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValZMM_13-CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/10824.5/index.html) for workflow 10824.5
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValZMM_13-CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/10824.8/index.html) for workflow 10824.8
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValZMM_13-CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/10824.7/index.html) for workflow 10824.7

### /RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - reference [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-74203703f0f911fed5d2697a665c5cd8177d84f2/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.7
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.7
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-74203703f0f911fed5d2697a665c5cd8177d84f2/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5%3A;referenceobj2=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference workflow 10824.5
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8%3A;referenceobj2=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.8
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7%3A;referenceobj2=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.7

### /RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - reference [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-74203703f0f911fed5d2697a665c5cd8177d84f2/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.7
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.7
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-74203703f0f911fed5d2697a665c5cd8177d84f2/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5%3A;referenceobj2=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference workflow 10824.5
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8%3A;referenceobj2=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.8
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7%3A;referenceobj2=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.7

makortel commented:
tracking validation links are still broken, DQM seem to work

(conceptually looks cool)

fwyzard commented:
Oops... actually, the links were mostly fine, but I had commented out the part that actually makes the plots :-D

fwyzard commented:
### /RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValTTbar_13-CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/10824.5/index.html) for workflow 10824.5
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValTTbar_13-CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/10824.8/index.html) for workflow 10824.8
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValTTbar_13-CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/10824.7/index.html) for workflow 10824.7

### /RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValZMM_13-CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/10824.5/index.html) for workflow 10824.5
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValZMM_13-CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/10824.8/index.html) for workflow 10824.8
  - [tracking validation plots](https://fwyzard.web.cern.ch/fwyzard/patatrack/pulls/57a96601916fe357a604a50ec0ebf5c07d6ab01f/RelValZMM_13-CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/10824.7/index.html) for workflow 10824.7

### /RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - reference [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-74203703f0f911fed5d2697a665c5cd8177d84f2/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.7
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.7
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-74203703f0f911fed5d2697a665c5cd8177d84f2/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5%3A;referenceobj2=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference workflow 10824.5
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8%3A;referenceobj2=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.8
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7%3A;referenceobj2=other%3A%3A/RelValTTbar_13/CMSSW_10_1_0_pre3-PU25ns_101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.7

### /RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1/GEN-SIM-DIGI-RAW
  - reference [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-74203703f0f911fed5d2697a665c5cd8177d84f2/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for development release, workflow 10824.7
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.5
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.8
  - [DQM plots](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=none;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for testing release, workflow 10824.7
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-74203703f0f911fed5d2697a665c5cd8177d84f2/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5%3A;referenceobj2=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_5%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for reference workflow 10824.5
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_8%3A;referenceobj2=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_8%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.8
  - [DQM comparison](http://dqmgui7.cern.ch:8060/dqm/dev/start?runnr=1;dataset=/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_5;sampletype=offline_relval;filter=all;referencepos=overlay;referenceshow=all;referencenorm=True;referenceobj1=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c0cd5117e0e553a4416324a73f5ac05f0621e901/10824_7%3A;referenceobj2=other%3A%3A/RelValZMM_13/CMSSW_10_1_0_pre3-101X_upgrade2018_realistic_v3-v1-c136348bca337c4754fcd9bacbdacd4b3b1b7b13/10824_7%3A;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Everything;size=M;root=Tracking;focus=;zoom=no;) for workflow 10824.7

fwyzard commented:
And the validation script itself now lives at https://github.com/cms-patatrack/patatrack-validation/blob/master/validate .
